### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by caseycesari**
Builds out the README by moving around some of the existing content into the appropriate sections, filling in some of the new required sections, as well as stubbing out some of the sections that still need to be developed. Those sections are *Creating a survey*, *Filling out a survey*, and *Survey administration*. Also, a brief overview of the OMK server installation is added.

### CODE REVIEW

File: README-EN.md [hunk 1]
```diff
@@ -1,64 +1,103 @@
...
 
-Administrators can access the site at
-* http://idb-omk.azavea.com/omk/pages/#/
+<img src="https://user-images.githubusercontent.com/1014341/50118987-639d3680-021f-11e9-910d-a0509d9b4f76.png" width="300px">
+
+Back on the app home screen, click the "Get Blank Form" button. This will download a list of all the forms available on the server. Once the list is populated, select the form you want and click "Get Selected". The form should now be downloaded to your device.
+
+From the home screen, click "Fill Blank Form", and select the survey you would like to complete. The survey should now launch. Swipe left to proceed through the survey questions.
+
+When filling out the survey, any geospatial questions will launch the OMK app, where you'll be able to add nodes with the attributes you specified in the survey form.
```
Comment (mmcfarland) [inline README-EN.md]: Can we mention here:
- "nodes" means only points in this context, can't add lines/polygons
- differentiate that attributes on the node features are different than survey questions. Or rather, they can both be specified in the survey, but some are questions and some are attributes on the OSM feature (at least that's my understanding)

File: README-EN.md [hunk 2]
```diff
@@ -1,64 +1,103 @@
...
+
+<img src="https://user-images.githubusercontent.com/1014341/50118981-6304a000-021f-11e9-9dbe-50393f6b5d03.png" width="300px">
+
+After you complete the geospatial components of the survey, you'll be redirected back to the ODK app.
+
+When you finish the survey, you need to send the results back to the OMK server. You can do this by clicking the "Send Finalized Form" button on the app home screen. Check any surveys that you have completed, and then click "Send Selected". The results of the survey will now be available on the server.
```
Comment (mmcfarland) [inline README-EN.md]: Add something about how this requires an internet connection.

File: README-EN.md [hunk 3]
```diff
@@ -1,64 +1,103 @@
 Visit the [Publication Guide](el-bid.github.io/guia-de-publicacion/) (currently available in Spanish) for more information about how to publish digital tools.
```
Comment (mmcfarland) [inline README-EN.md]: Oh, also, I wasn't sure about this but ended up deleting it from the other IDB repo. I think it's part of the template.

Comment (caseycesari) [reply README-EN.md]: Yeah, I wasn't sure either. It didn't seem like guidance for the filling out the README, so I thought it was meant to stay

**Comment by caseycesari** [comment]
Pushed up some changes.

**Pull request merged**